### PR TITLE
feat(language): added support for dutch nl-SR language and currency

### PR DESCRIPTION
**Pull request by blue-predator**
Please merge and release

### CODE REVIEW

**Comment by blue-predator** [comment]
Seems 3 other languages test are not complete.
image

File: src/ToWords.ts [hunk 1]
```diff
@@ -38,8 +36,6 @@ export class ToWords {
   public getLocaleClass(): ConstructorOf<LocaleInterface> {
     /* eslint-disable @typescript-eslint/no-var-requires */
     switch (this.options.localeCode) {
-      case 'en-BD':
-        return enBd;
```
Comment (mastermunj) [inline src/ToWords.ts:42]: @blue-predator These changes could be the reason for test cases failing for en-BD, en-GB and tr-TR.

File: src/ToWords.ts [hunk 2]
```diff
@@ -231,7 +225,7 @@ export class ToWords {
     if (quotient > 1 && locale.config?.pluralWords?.find((word) => word === match.value) && locale.config?.pluralMark) {
       matchValue += locale.config.pluralMark;
     }
-    if (quotient === 1 && locale.config?.ignoreOneForWords?.includes(matchValue)) {
+    if (quotient === 1 && locale.config?.ignoreOneForWords) {
```
Comment (mastermunj) [inline src/ToWords.ts:228]: This also doesn't seem a needed change.

**Review by mastermunj — CHANGES_REQUESTED** [review]

**Comment by blue-predator** [comment]
> Seems 3 other languages test are not complete. image

Sorry it was my mistake, I'm rushing too much on this. I had overwritten the ToWords.ts with the one from v3 thinking they where the same. All is good now.
image

**Review by mastermunj — APPROVED** [review]

**Comment by mastermunj** [comment]
Thank you. Released 3.2.0.

**Pull request merged**